app/models/manga.rb

```diff
@@ -86,16 +86,31 @@ def canonical_title
     romaji_title
   end
 
-  def poster_image_thumb
+  def poster_image_thumb(size = :large)
```

Comment:
I thought so… I wasn't sure to refactor so much since it was my first time working in the code but I'll make a proper refactor when I finish the api search :)